### DISCUSSION TITLE
[Docs] Is there any plan to synchronize Chinese documents with English documents?

### DISCUSSION THREAD

**Discussion by Sefank** [thu-ml/tianshou#960] [Ideas]
Currently, it seems that Chinese documents have stopped updating, and many of the necessary instructions in English documents are completely missing in Chinese documents, which confuses newcomers like me a lot. The inconsistency between English and Chinese documents is so weird for me.

If it is not intended, I would like to give it a shot to help synchronize English documents into Chinese documents.

**Comment by Trinkle23897** [comment]
Feel free to submit a pr!
